### ISSUE TITLE
app crash on multiple barcode

### ISSUE THREAD

**Issue by mhdaslam790** [mohesu/barcode_scanner#25]
App crashes on multiple barcode .Whenever there is a more than one barcode in front of camera the app gets crashed.

**Comment by rvndsngwn** [comment]
trying to fix in next release.

**Comment by rvndsngwn** [comment]
This issue fixed in [0.0.4 version](https://pub.dev/packages/ai_barcode_scanner)

**Comment by rvndsngwn** [comment]
```
      controller: MobileScannerController(
        detectionSpeed: DetectionSpeed.noDuplicates,
      ),
```

add controller in you code.
